OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -414,6 +421,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </decoNote>
 
                         </decoDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="Unclear">On the leather fold on the inside of the cover, there are a few barely legible letters in 
```

Comment:
I would reformulate: There are a few barely legible ... on the leather fold on ....

Reply:
(or even "A few barely ....on the leather fold...")

Reply:
Yes, I think this depends on the standardization of the syntax during the revision process. We'll see if it is more convenient to remove verbs everywhere or to add them and make full sentences everywhere.